src/dashboard/connector.ts

```diff
@@ -0,0 +1,19 @@
+export default class DashboardConnector {
+    public async sendEmail (email: string): Promise<any> { // eslint-disable-line @typescript-eslint/no-unused-vars
+        return {
+            success: true,
+            message: 'Email sent',
+        };
+    }
+
+    public async validateToken (token: string): Promise<any> { // eslint-disable-line @typescript-eslint/no-unused-vars
+        return {
+            valid:   true,
+            message: 'ok',
+        };
+    }
+
+    public async getDefaultProjectLink (token: string): Promise<string> { //eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
This request is redundant. Link to the project looks like this `https://testcafe.dashboard.com/${projectId}`. You can get `projectId` from token: https://github.com/DevExpress/testcafe-reporter-dashboard/blob/master/src/validate-settings.ts#L29